packages/dev/core/src/Rendering/IBLShadows/iblShadowsVoxelTracingPass.ts

```diff
@@ -323,17 +324,15 @@ export class _IblShadowsVoxelTracingPass {
         // Need to set all the textures first so that the effect gets created with the proper uniforms.
         this._setBindings(this._scene.activeCamera!);
 
-        let counter = 0;
-        this._scene.onBeforeRenderObservable.add(() => {
-            counter = 0;
-        });
-        this._scene.onAfterRenderTargetsRenderObservable.add(() => {
-            if (++counter == 2) {
-                if (this.enabled && this._outputTexture.isReady()) {
-                    this._setBindings(this._scene.activeCamera!);
-                    this._outputTexture.render();
+        // Don't start rendering until the first vozelization is done.
+        this._renderPipeline.onVoxelizationCompleteObservable.addOnce(() => {
+            this._scene.geometryBufferRenderer!.getGBuffer().onAfterRenderObservable.add(() => {

```

Comment:
Same and sorry again I did not notice in the past.

Reply:
Done